### ISSUE TITLE
Fix multiline text parser problem

### ISSUE THREAD

**Issue by agershun** [agershun/alasql#257]
```js
var res = alasql(function(){/*
     CREATE TABLE one (
                t TEXT
     );
     INSERT INTO one VALUES ('My  
     huge * * * * * * * * * * * * * ************* ****** ****** text');
    SELECT * FROM one;
*/});
```

See [this example](http://jsfiddle.net/agershun/bhbtf86u/1/). 

It does not work, because AlaSQL parser does not recognize multi-line string inside SQL text.

**Comment by mathiasrw** [comment]
I dont get this. 

The SQL is in a comment so it corresponds to 

    var res = alasql(function(){});

so no output...

**Comment by agershun** [comment]
Yes, there is no output because there is a line break inside the text string. Try to remove it, and all will work. Plus, AlaSQL does not support backslashed characters like "\t"

Отправлено с iPhone

**Comment by mathiasrw** [comment]
OK ok - I get it. Was not aware of this notation...
